administrator/components/com_patchtester/script.php

```diff
@@ -53,22 +53,22 @@ public function postflight($type, $parent)
 	{
 		if ($type == 'install')
 		{
-				$language = JFactory::getLanguage();
-				$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
-				$language->load('com_patchtester', JPATH_SITE, null, true);
-				echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
-				echo '<p>' . Text::_('COM_PATCHTESTER_XML_DESCRIPTION') . '</p>';
-				echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_patchtester">' . Text::_('COM_PATCHTESTER') . '</a></p>';
-				echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication	&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
+			$language = JFactory::getLanguage();
+			$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
+			$language->load('com_patchtester', JPATH_SITE, null, true);
+			echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_XML_DESCRIPTION') . '</p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_patchtester">' . Text::_('COM_PATCHTESTER') . '</a></p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication	&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
```

Comment:
```suggestion
			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
```
Please again remove the tabulator from the URL of the href.

Reply:
Was maybe my mistake that it came back.

Reply:
And line 62 is too long, that is still the final PHP code style error. But that makes me ask: Why is there this hard-coded key in the URL? This is what makes it so long, and it doesn't look right to me.

Reply:
@roland-d Could you have a look on that?

Reply:
The key can be removed, think it makes it load the component after save

Reply:
They key looks like a dynamically created key to me, and if that is true then it definitely can't be hardcoded,

Reply:
And there is a "&path=" before that key without any value, that seems also strange to me.